pages/gameboard/index.js

```diff
@@ -16,7 +16,7 @@ export default function Gameboard() {
 
   useEffect(() => {
     if (!singleCategories.length) {
-      router.push("/");
+      window.location.href = "/";
```

Comment:
You made this change here but not in the individual question pages. Can you explain why?

Reply:
Good catch will change it in questions too. Using `router.push` creates odd behavior and read some suggestions to use `window.location.href`